src/models/vtr/classifier.py

```diff
@@ -62,9 +69,11 @@ def __init__(
         self.encoder = nn.TransformerEncoder(encoder_layer, num_layers=num_layers)
         self.norm = nn.LayerNorm(hidden_size)
         self.classifier = nn.Linear(hidden_size, 1)
+        self.ocr = BiLSTM(input_size=256, hidden_size=256, num_layers=2, num_classes=60)
```

Comment:
Let's not hard-code all these numbers, add them to class parameters in any form so in the future we can easily change them.